bioptim/optimization/optimal_control_program.py

```diff
@@ -743,7 +743,326 @@ def load(file_path: str) -> list:
+                               color='black')
+                        if len(main_nodes) > 1:
+                            for idx in range(1, len(main_nodes)):
+                                G.edge(f'node_struct_{phase_idx}{main_nodes[idx-1]}',
+                                       f'node_struct_{phase_idx}{main_nodes[idx]}',
+                                       color='black')
+
+                # Draw edges between nlp node and parameters
+                def draw_nlp_to_parameters_edges():
+                    nb_parameters = len(ocp.nlp[phase_idx].parameters)
+                    G.edge(f'nlp_node_{phase_idx}',
+                           f'param_{phase_idx}0',
+                           color='black')
+                    for param_idx in range(nb_parameters):
+                        if param_idx >= 1:
+                            G.edge(f'param_{phase_idx}{param_idx - 1}',
+                                   f'param_{phase_idx}{param_idx}',
+                                   color='black')
+                    if nb_parameters > 1:
+                        G.edge(f'param_{phase_idx}{nb_parameters-1}',
+                               f'lagrange_{phase_idx}',
+                               color='black')
+                    else:
+                        G.edge(f'param_{phase_idx}0',
+                               f'lagrange_{phase_idx}',
+                               color='black')
+
+                # Draw edges between OCP node and each nlp cluster
+                G.edge('OCP', f'nlp_node_{phase_idx}')
+
+                draw_nlp_to_parameters_edges()
+                draw_shooting_nodes_edges()
+
+            # Display phase transitions
+            def display_phase_transitions():
+                with G.subgraph(name=f'cluster_phase_transitions') as g:
+                    g.attr(style='', color='black')
+                    g.node_attr.update(style='filled', color='grey')
+                    for phase_idx in range(self.n_phases):
+                        if phase_idx != self.n_phases - 1:
+                            g.node(f'Phase #{phase_idx}')
+                            g.node(f'Phase #{phase_idx + 1}')
+                            g.edge(f'Phase #{phase_idx}',
+                                   f'Phase #{phase_idx + 1}',
+                                   label=ocp.phase_transitions[phase_idx].type.name)
+                    g.attr(label=f"Phase transitions")
+
+            # Draw OCP node
+            G.node('OCP', shape='Mdiamond')
+
+            # Draw nlp clusters and edges
+            for phase_idx in range(ocp.n_phases):
+                main_nodes = draw_nlp_cluster(phase_idx)
+                draw_edges(main_nodes)
+
+            # Draw phase_transitions
+            display_phase_transitions()
+
+            # Display graph
+            G.view()
+
+        if to_console:
+            print_console(self)
+
+        if to_graph:
+            draw_graph(self)
+
+    def __define_time(
```

Comment:
Make sure it is not supposed to be `_define_time()`

Reply:
I corrected this mistake. Indeed, it was supposed to be _define_time()

Reply:
Done